ofl/roboto/OFL.txt

```diff
@@ -1,8 +1,8 @@
-Copyright 2011 The Roboto Project Authors (https://github.com/google/roboto)
+Copyright 2011 Google Inc. All Rights Reserved.
 
 This Font Software is licensed under the SIL Open Font License, Version 1.1.
 This license is copied below, and is also available with a FAQ at:
-http://scripts.sil.org/OFL
+https://scripts.sil.org/OFL
```

Comment:
Following https://github.com/google/fonts/issues/7062 the OFL url should be updated to https://openfontlicense.org/